### ISSUE TITLE
Land and property investments

### ISSUE THREAD

**Issue by BobHarper1** [SocialInvestmentDataLab/spec#8]
Looking at [BSC's deal level data (Dec 2016)](http://www.bigsocietycapital.com/sites/default/files/Social%20Investment%20Activity%20Dec%202016%20FINAL.xlsx)

They categorise property as a form of investment. More specifically, it might be a type of financing that enables property or land purchase, as the fields make provision for a 'deal type if property' (options: purchase and leaseback, or build and leaseback).

Implications for a data spec:
1. Is property a type of investment (forming part of a deal) itself?
2. Is a property a form of credit (which it usually is), and if so, should there be a way to denote this within Credit.

I'd prefer the latter. My understanding is that property deals in social finance are essentially an issue of credit (lease) to allow the continued use of property by the investee (often a housing association) which they don't own.

**Comment by ScatteredInk** [comment]
I think we need to distinguish between (1) cases where credit is being provided to _acquire_ or _build_ an asset and (2) cases where an existing asset is being sold to release capital.

Cases in (1) fit with the credit model and would cover a normal loan to buy/purchase a building. We could then use a `purpose`-type classification to get to the building purchase. The build-to-leaseback model _could_ fit this model but I think we need to know more about who is taking on the credit, and the credit risk, before the asset transfer takes place (assuming that leaseback has the common meaning, here, of the building ending up with a landlord/creditor who leases it back to the occupant).

Cases in (2), like sale-and-leaseback, don't fit the credit model. If an asset is sold and leased back then that's a commercial agreement post-purchase, rather than credit. 

There is, I think, a strong argument for the data spec eventually covering asset sales as the need/ability to create liquidity or release capital looks a useful metric to track. We'd benefit from some domain expertise on this issue also.

**Comment by ScatteredInk** [comment]
From conversation with @Genmh, we may want to separately:

1. track property purchases as the `purpose` of a deal.
2. associate a property `asset` with a deal or organisation.
3. associate indicators (planned/actual) with an asset.

**Comment by Genmh** [comment]
Given the mortgage data in companies house, does it also make sense to track secured v. unsecured loans? The surveys of socinv always make a great deal of the need for unsecured loans, but in the KF data there's often an unsecured loan that is subsequently 'secured' with a lien on an acquired asset. Does that make sense?!

**Comment by ScatteredInk** [comment]
> The surveys of socinv always make a great deal of the need for unsecured loans, but in the KF data there's often an unsecured loan that is subsequently 'secured' with a lien on an acquired asset. Does that make sense?!

@Genmh So is this a variation of an existing loan? Is the timeline: get unsecured loan -> get asset -> loan is 'secured' - or am I misunderstanding?

**Comment by Genmh** [comment]
@ScatteredInk yep, that's it. But not a variation precisely, in that this is usually agreed as part of the initial loan.
